apps/aehttp/priv/swagger.json

```diff
@@ -2280,12 +2469,21 @@
         "fee" : {
           "type" : "integer",
           "format" : "int64"
+        },
+        "account" : {
+          "$ref" : "#/definitions/EncodedHash"
+        },
+        "nonce" : {
+          "type" : "integer",
+          "format" : "int64"
         }
       },
       "example" : {
         "name_salt" : 0,
         "fee" : 6,
-        "name" : "name"
+        "name" : "name",
+        "nonce" : 1,
+        "account" : { }
```

Comment:
Should we have real sample hash as example account? 🤔

Reply:
We should define first so swagger-codegen could actually generate it...